x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/import_rule_exceptions.ts

```diff
@@ -43,7 +43,7 @@ export const importRuleExceptions = async ({
   const {
     errors,
     success,
-    success_count: successCount,
+    success_count_exception_list_items: successCount,
```

Comment:
It would be nice to have a comment why `success_count_exception_list_items` is used instead of `success_count`. Otherwise it may raise questions.